### PR TITLE
Doc updates

### PR DESCRIPTION
**Pull request by cane-lagerwaard**
TODO:

- [ ] update links

### CODE REVIEW

File: docs/backend/installing-magento2.md [hunk 1]
```diff
@@ -12,7 +12,8 @@ DEITY Magento module is compatible with Magento version 2.2. CE and EE versions.
 Full scale support of versions 2.0.* and 2.1.* is not our priority.
 However if you encounter an issue running with Magento version lower than 2.2 feel free
 to open an issue or reach out to our support channel.
-```
+
+```text
 magento > 2.2
```
Comment (sergy-gzh) [inline docs/backend/installing-magento2.md]: @cane-lagerwaard magento 2.2 is supported, can you change this to `>=`

File: website/sidebars.json [hunk 2]
```diff
@@ -1,11 +1,45 @@
...
+      "falcon-client/code-splitting",
+      "falcon-client/state-management",
+      "falcon-client/internationalization",
+      "falcon-client/falcon-ui"
+    ],
+    "Falcon Server": [
+      "falcon-server/basics",
+      "falcon-server/apicontainer",
+      "falcon-server/extensioncontainer",
+      "falcon-server/dynamic-route-resolver",
```
Comment (Borales) [inline website/sidebars.json]: @cane-lagerwaard this one could be merged into "falcon-server/basics",

File: website/sidebars.json [hunk 3]
```diff
@@ -1,11 +1,45 @@
...
+      "falcon-server/schema-stitching",
+      "falcon-server/shop-extension",
+      "falcon-server/blog-extension",
+      "falcon-server/falcon-magento2-api",
+      "falcon-server/falcon-wordpress-api",
```
Comment (Borales) [inline website/sidebars.json]: @cane-lagerwaard @mateuszwozniak just a thought.

- "falcon-server/shop-extension" and "falcon-server/blog-extension" -> "Extensions" group
- "falcon-server/falcon-magento2-api" and "falcon-server/falcon-wordpress-api" -> "API Provides" group

They have simple instructions, nothing special

Comment (mateuszwozniak) [reply website/sidebars.json]: Yes, I think that makes sense especially when we start adding more of those.

File: docs/backend/installing-magento2.md (hunk 1 above)
Comment (cane-lagerwaard) [reply docs/backend/installing-magento2.md]: @sergy-gzh can you make a suggestion in github? :)

File: docs/backend/installing-magento2.md [hunk 4]
```diff
@@ -12,7 +12,8 @@ DEITY Magento module is compatible with Magento version 2.2. CE and EE versions.
 Full scale support of versions 2.0.* and 2.1.* is not our priority.
 However if you encounter an issue running with Magento version lower than 2.2 feel free
 to open an issue or reach out to our support channel.
-```
+
```
[the remark this answers is not in the document]
Comment (cane-lagerwaard) [reply docs/backend/installing-magento2.md]: @Borales we could, please make a github suggestion

File: docs/backend/installing-magento2.md (hunk 1 above)
Comment (sergy-gzh) [reply docs/backend/installing-magento2.md]:
```suggestion
magento >= 2.2
```

File: docs/backend/installing-magento2.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (Borales) [reply docs/backend/installing-magento2.md]: @cane-lagerwaard I'll do it after Sergii's one

[the remark this answers is not in the document]
Comment (Borales) [reply docs/backend/installing-magento2.md]: @cane-lagerwaard I actually removed it completely, we described Magento version compatibility above this line, so I don't think we need it.

File: website/sidebars.json (hunk 3 above)
Comment (cane-lagerwaard) [reply website/sidebars.json]: @Borales @mateuszwozniak makes sense

**Pull request merged**